packages/api-client/src/components/EnvironmentSelector/EnvironmentSelector.vue

```diff
@@ -43,7 +43,7 @@ const selectedEnvironment = computed(() => {
   return (
     environment?.uid ||
     collection?.['x-scalar-active-environment'] ||
-    'No Environment'
+    'Environment'
```

Comment:
should we maintain `No` here? so we know that nothing is set?

Reply:
It's cleaner with dropping that just in the empty state, if it's a strong requirement than can add back